templates/dev/scripts/create-dao.js

```diff
@@ -15,13 +14,13 @@ const settings = {
   // The order is important
   dotVotingSettings: [ SUPPORT_REQUIRED, MIN_ACCEPTANCE_QUORUM, VOTE_DURATION ],
   financePeriod: 0,
-  id: randomId(),
+  id: '',
```

Comment:
Is this required for the e2e testing? it seems to be messing our current development flow

Reply:
I wrote a script that sequentially generates dao id's here: https://github.com/AutarkLabs/open-enterprise/blob/3b65f63b1d400d5cebf553fb21b93b573803809f/templates/dev/scripts/getId.js

The param in the settings above will serve as an override if a dev wants to set their own ID